pkg/apis/core/fuzzer/fuzzer.go

```diff
@@ -309,6 +309,23 @@ var Funcs = func(codecs runtimeserializer.CodecFactory) []interface{} {
 			c.FuzzNoCustom(ct)                                          // fuzz self without calling this function again
 			ct.TerminationMessagePath = "/" + ct.TerminationMessagePath // Must be non-empty
 			ct.TerminationMessagePolicy = "File"
+			// Match defaulting in pkg/apis/core/v1/defaults.go.
+			_, hasCPUReq := ct.Resources.Requests[core.ResourceCPU]
```

Comment:
Given that this is a fuzzer, the chances of this happening is suuuuuuper slim.

Perhaps we should add something like:

* pick a random bool, if true set requests.cpu to a random value value between 100 and 10000 + "m"
* pick a random bool, if true set limits.cpu to prev.request + a random value between 0 and 1
* repeat for memory

That will give us a good probability of exercising the resources path.

Then here, set the policy to `[]string{ core.NotRequired, core.Required }[random int between 0 and 1]`
?